### PR TITLE
Improvements in Planning interface port to ROS 2

### PR DESCRIPTION
**Pull request by ahcorde**

### CODE REVIEW

File: moveit_core/planning_interface/include/moveit/planning_interface/planning_interface.h [hunk 1]
```diff
@@ -165,7 +165,7 @@ class PlannerManager
   /// It is assumed that motion plans will be computed for the robot described by \e model and that any exposed ROS
   /// functionality
   /// or required ROS parameters are namespaced by \e ns
-  virtual bool initialize(const robot_model::RobotModelConstPtr& model, const std::string& ns);
+  virtual bool initialize(const robot_model::RobotModelConstPtr& model, std::shared_ptr<rclcpp::Node>& node);
```
Comment (mlautman) [inline moveit_core/planning_interface/include/moveit/planning_interface/planning_interface.h]: Are you sure we will no longer need the namespace? Also, can you point me to some code where you use the node?

File: moveit_core/planning_interface/include/moveit/planning_interface/planning_interface.h [hunk 2]
```diff
@@ -165,7 +165,7 @@ class PlannerManager
   /// It is assumed that motion plans will be computed for the robot described by \e model and that any exposed ROS
   /// functionality
   /// or required ROS parameters are namespaced by \e ns
```
Comment (mlautman) [inline moveit_core/planning_interface/include/moveit/planning_interface/planning_interface.h]: Update the docstrings where appropriate

Comment (ahcorde) [reply moveit_core/planning_interface/include/moveit/planning_interface/planning_interface.h]: I think is the only header that has changed

Comment (ahcorde) [reply moveit_core/planning_interface/include/moveit/planning_interface/planning_interface.h]: friendly ping @mlautman

**Comment by ahcorde** [comment]
@mlautman @henningkayser  @davetcoleman Ready to merge

**Pull request merged**